README.md

```diff
@@ -2,7 +2,9 @@
 
 W3C XML Encryption implementation for node.js (http://www.w3.org/TR/xmlenc-core/)
 
-Supports node >= 12
+Supports node >= 12 < 18
```

Comment:
We should address the Triple DES issues instead of dropping support for Node 18. We likely don't need to support that algorithm at all

Reply:
Agreed. This is more for backwards compatibility reasons. A drop in algorithm will require a major version bump.